roles/core/log_client/templates/rsyslog.conf.j2

```diff
@@ -16,11 +16,11 @@ $ModLoad imjournal # provides access to the systemd journal
 
 # Provides UDP syslog reception
 #$ModLoad imudp
-#$UDPServerRun 514
+#$UDPServerRun {{ log_client_rsyslog_port }}
```

Comment:
Sorry, not here 🙂 
You should add this variable to line 93 of the file. See my precedent post.
This part here is only a comment (see the # at beginning of line), that could even be removed to clean the template (this is a relic of devs).